defaults/main/main_vars.yml

```diff
@@ -0,0 +1,58 @@
+---
+# task-specific includes
+
+# sshconfig.yml
+arh_overwrite_ssh_config: true
+arh_overwrite_sshd_config: true
+arh_set_sshd_host_key_permissions: true
+
+# users.yml
+arh_modify_adduser_conf: true
+arh_overwrite_useradd: true
+
+
+# include top level vars in tasks/main.yml
```

Comment:
I think you need to group files logically instead of going thru the yaml files, e.g. `include_packagemgmt: false` will break things and without `include_packages: true` multiple tasks will fail as well since configuration files etc are dependent on this.

Reply:
I see that packagememgt is similar to the requirements set up in `pre.yml`. It's also easy to hold back the packagemgmt.yml from the conditionals.

I could present the individual play vars in one file, grouped loosely by logical focus area. Is something like that what you're thinking?

Reply:
Yeah, exactly, much like using the existing tags and group those